### PR TITLE
Add default field to subscriptions and prettify subscriptions

### PR DESCRIPTION
**Pull request by Elblinator**
unline the PR #4236

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts [hunk 1]
```diff
@@ -61,14 +66,16 @@ export const getProjectorListSubscriptionConfig: SubscriptionConfigGenerator = (
...
                 ]
             },
-            `projector_countdown_ids`,
-            `projector_message_ids`,
-            ...MEETING_DEFAULT_PROJECTOR_IDS_KEYS,
-            { idField: `speaker_ids`, additionalFields: [`meeting_user_id`] },
-            `list_of_speakers_ids`,
-            { idField: `agenda_item_ids`, fieldset: [`item_number`, `content_object_id`] }
-        ],
-        additionalFields: [`reference_projector_id`]
```
Comment (reiterl) [inline client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts:71]: This seems to be missing.

**Review by reiterl — COMMENTED** [review]
At one place an `additionalFields` is gone. Please have a look at it.

File: client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts (hunk 1 above)
Comment (Elblinator) [reply client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts:71]: I moved it to line 28

**Review by reiterl — APPROVED** [review]
Looks okay to me.

**Review by bastianjoel — CHANGES_REQUESTED** [review]
First of all you should use `FULL_FIELDSET`. 

Besides from that using the `FULL_FIELDSET` in most cases is not the correct way to go! For example in `projectors.subscription.ts` you are now subscribing the full meeting information. This is unnecessary. These warnings are there for a reason. Just suppressing them without thinking about if you really need all fields of a subscribed collection makes it unnecessarily harder for us in the future to fix these occurrences.

File: client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts (hunk 1 above)
Comment (bastianjoel) [reply client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts:71]: `additionalFields` was actually unnecessary. You don't need to subscribe additional fields when already subscribing ALL fields.

**Pull request closed without merging**